DashKit/PodAlert.swift

```diff
@@ -110,10 +110,8 @@ public enum PodAlert: String {
     
     var actionButtonLabel: String {
         switch self {
-        case .suspendEnded:
-            return LocalizedString("OK", comment: "Action button text to acknowledge pump resume")
         default:
-            return LocalizedString("Acknowledge", comment: "Action button default text for PodAlerts")
+            return LocalizedString("Ok", comment: "Action button default text for PodAlerts")
```

Comment:
Do you need to keep the switch statement?

Reply:
Not really. Keep thinking we'll need it, but I'll clean this up for now.